examples/swe_agent/main.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example with SWE-agent\n",
```

Comment:
Please change the title to "Conversation with Software Engineering Agent". The same within README.md